src/frameworks.sources

```diff
@@ -955,6 +955,11 @@ INTENTS_SOURCES = \
 	Intents/INRelativeReferenceResolutionResult.watchOS.cs \
 	Intents/INRelativeSettingResolutionResult.watchOS.cs \
 
+# IntentsUI
+
```

Comment:
nit: looks like this might be an extra line

Reply:
Yes maybe, but moreover I didn't need to edit this file. I misread the diff